fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/ClientSavingsIntegrationTest.java

```diff
@@ -257,22 +257,22 @@ public void testSavingsAccount_WITH_ENFORCE_MIN_BALANCE() {
 
         DateFormat dateFormat = new SimpleDateFormat("dd MMMM yyyy");
         Calendar todaysDate = Calendar.getInstance();
-        final String TRANSACTION_DATE = dateFormat.format(todaysDate.getTime());
+        final String transactionDateValue = dateFormat.format(todaysDate.getTime());
```

Comment:
This probably is a side-effect of mass find-replace, thank you for finding this, updating.

Reply:
I actually think this was right (good), because all upper case variable names (on Java), by convention, are ONLY for true static final constants (which to this is not)..

Reply:
@vorburger  in that case:
This would clash with AbbreviationAsWordInName checkstyle which only allows up to 4 capital letters in final variables. 
What should we go with?

Reply:
Ok agreed, making the changes.